main.js

```diff
@@ -0,0 +1,83 @@
+var tbody = document.querySelector('tbody');
+var btn = document.querySelector('button');
+var expName = document.querySelector('input[name="exp-name"]');
+var expAmount = document.querySelector('input[name="exp-amount"]');
+var expTotal = document.querySelector(".exp-total");
+var total = 0;
+var removeButtons = [];
+var expense;
+
+
+var renderRow = function(name, amount) {
+  var tr = document.createElement('tr');
+  var td = document.createElement('td');
+  var tbody = document.querySelector('tbody')
+  var tdRemove = document.createElement('td');
+  var rmv = document.createElement('button');
+  rmv.textContent = 'Remove';
+  rmv.classList = 'remove';
+  td.innerHTML = name;
+  tr.appendChild(td);
+  td = document.createElement('td');
+  td.textContent = amount;
+  tr.appendChild(td);
+  tr.appendChild(tdRemove);
+  tdRemove.appendChild(rmv);
+  removeButtons.push(rmv);
```

Comment:
Rather than adding the remove buttons to an array and looping through it, you can just add the event listener to the remove button you're creating at that time. 
`rmv.addEventListener('click', handleRemove)` That'll always work.